src/sql/migrations.rs

```diff
@@ -622,6 +622,10 @@ impl Sql {
         .await
         .with_context(|| format!("execute_migration failed for version {}", version))?;
 
+        let mut lock = self.config_cache.write().await;
+        lock.insert(VERSION_CFG.to_string(), Some(format!("{}", version)));
+        drop(lock);
```

Comment:
what's this for?

Reply:
it makes sure, the cache is in sync with the database; some lines above, the config value is set (inside the transaction not using the "normal" `set_config_raw()`)